firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2015,6 +2015,15 @@ class BrowserViewController: UIViewController,
             }
 
         case .URL:
+            if isSwipingTabsEnabled {
```

Comment:
Should we maybe check if `shouldAnimate` is false as well to avoid an extra dispatch? Not sure if it's worth it though.